memgpt/client/client.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict, List, Union
+
+from memgpt.config import AgentConfig
+from memgpt.persistence_manager import PersistenceManager
+from memgpt.server.rest_api.interface import QueuingInterface
+from memgpt.server.server import SyncServer
+
+
+class Client(object):
```

Comment:
Not too sure about the name `Client`.  It sort of implies it will be connecting to a rest or ws server.  Maybe `LocalClient`?